README.md

```diff
@@ -1,57 +1,60 @@
-# RIF Enveloping
+# RIF Enveloping - V2
 
-Secure sponsored transaction system to enable users to pay fees using ERC-20 tokens.
+A secure sponsored-transaction system to enable users to pay fees using ERC-20 tokens.
 
 [![CircleCI](https://circleci.com/gh/rsksmart/enveloping/tree/master.svg?style=shield)](https://circleci.com/gh/rsksmart/enveloping/tree/master)
+## Description
 
-## Table of Contents
+This version is based on the [Gas Station Network (GSN) project](https://github.com/opengsn/gsn). The GSN is a decentralized system that improves dApp usability without sacrificing security. In a nutshell, GSN abstracts away gas (used to pay transaction fees) to minimize onboarding & UX friction for dApps. With GSN, "gasless clients" can interact with Ethereum contracts without having to acquire ETH to pay for gas.
```

Comment:
We should rephrase this part to state that V1 was fully based on GSN, however V2 is a redesign.